src/configurator/v1.ml

```diff
@@ -520,6 +520,14 @@ module Pkg_config = struct
             let prefix =
               String.trim (Process.run_capture_exn c ~dir brew ["--prefix"])
             in
+            let package =
+              let seps = ['='; '>'; '<'] in
+              match List.filter_map ~f:(String.index package) seps with
+              | [] -> package
+              | init :: xs ->
+                let pos = List.fold_left ~init ~f:min xs in
+                String.trim (String.take package pos)
+            in
             [sprintf "PKG_CONFIG_PATH=$PKG_CONFIG_PATH:%s/opt/%s/lib/pkgconfig"
```

Comment:
Wouldn't it be better to construct `%s/opt/%s/lib/pkgconfig` and quote it?

Reply:
Quoting the environment is a good idea (Although I have a better fix for that coming up), but I don't think it would fix the issue. The `~package` contains a pkg-config query which contains more than just name + version. So it wouldn't work to include it in the path.